src/kafka/account-generator/AccountGenerator.ts

```diff
@@ -24,19 +24,19 @@ export class AccountGenerator {
    *
    * @param producer - Producer used to interact with Orchestrate
    */
-  constructor(brokers: string[]) {
-    this.producer = new Producer(brokers, {
-      logLevel: logLevel.NOTHING,
-      clientId: 'wallet-generator-sdk-producer'
+  constructor(
+    brokers: string[],
+    kafkaConfig?: Omit<KakfaJS.KafkaConfig, 'brokers'>,
+    producerConfig?: KakfaJS.ProducerConfig,
+    consumerConfig?: KakfaJS.ConsumerConfig
+  ) {
+    this.producer = new Producer(brokers, { clientId: 'wallet-generator-sdk-producer', ...kafkaConfig }, producerConfig)
+
+    this.consumer = new Consumer(brokers, [DEFAULT_TOPIC_ACCOUNT_GENERATED], kafkaConfig, {
+      groupId: 'wallet-generator-sdk-consumer',
```

Comment:
IMO we should use `logLevel: logLevel.NOTHING,` as default value of the default cfg